tapdb/migrations_test.go

```diff
@@ -330,24 +331,25 @@ func TestMigration29(t *testing.T) {
 	// Check universe_roots: the dummy row should keep its original
 	// proof_type.
 	var proofType string
-	err = db.QueryRowContext(ctx,
-		"SELECT proof_type FROM universe_roots WHERE id = 1").Scan(
-		&proofType,
-	)
+	err = db.QueryRowContext(ctx, `
+		SELECT proof_type FROM universe_roots WHERE id = 1
+	`).Scan(&proofType)
 	require.NoError(t, err)
 	require.Equal(t, "issuance", proofType)
 
 	// Check federation_global_sync_config dummy data.
-	err = db.QueryRowContext(ctx,
-		`SELECT proof_type 
```

Comment:
I think this was actually my inconsistency lol. Got review feedback to limit the line length on the queries, and just kinda hacked away at it until things were in line.